src/cli/global/upgrade.rs

```diff
@@ -30,6 +30,10 @@ pub struct Args {
     /// The platform to install the package for.
     #[clap(long, default_value_t = Platform::current())]
     platform: Platform,
+
+    /// Do not insert conda_prefix, path modifications, and activation script into the installed executable script.
+    #[clap(long)]
+    no_activation: bool,
```

Comment:
Let's only add this flag to `pixi global install`

Reply:
not sure how you would upgrade a no activation package without this option. maybe that will only be possible in the new format of global install.

Reply:
Yeah, fair point. Let's add the flag to the two update commands for now then.